test/Recipe-test.js

```diff
@@ -0,0 +1,113 @@
+import { expect } from 'chai';
+import Recipe from '../src/classes/Recipe';
+
+describe('Recipe', () => {
+    it('Should be a function', () => {
+      expect(Recipe).to.be.a('function');
+    });
+
+let recipe1;
+let recipe2;
+let recipe3; 
+let recipe4;   
+
+ beforeEach(() => {
```

Comment:
lines 14-23 should be indented differently. beforeEach is a child of the describe block, and the variable reassignments are children of the beforeEach's callback argument

Reply:
Formatting was corrected, and pushed up